doc/modules/qualitymetrics/firing_range.rst

```diff
@@ -0,0 +1,40 @@
+Firing range (:code:`firing_range`)
+===================================
+
+
+Calculation
+-----------
+
+The firing range indicates the dispersion of the firing rate of a unit across the recording. It is computed by
+taking the difference between the 95-th and 5th percentiles firing rates computed over short time bins (e.g. 10 s).
+
+
+
+Expectation and use
+-------------------
+
+Very high levels of firing ranges, outside of a physiolocigal range, might indicate noise contamination.
+
+
+Example code
+------------
+
+.. code-block:: python
+
+    import spikeinterface.qualitymetrics as qm
+
+    # Make recording, sorting and wvf_extractor object for your data.
+    firing_range = qm.compute_firing_ranges(wvf_extractor)
+    # firing_range is a dict containing the units' IDs as keys,
```

Comment:
```suggestion
    # firing_range is a dict containing the unit IDs as keys,
```

I think the meaning is that each key is a different unit ID, in which case you would say unit IDs. The way it is written means that each unit has multiple IDs that are being used as keys, but maybe I'm misunderstanding what data is being returned?

Reply:
done for all quality metrics in last commit